openbb_terminal/stocks/options/yfinance_view.py

```diff
@@ -45,6 +46,15 @@
 logger = logging.getLogger(__name__)
 
 
+def header_fmt(header: str) -> str:
```

Comment:
Docstring plz

Reply:
Ah shoot already merged. Ill throw it in the other one